src/app/public/components/badge-assertion/badge-assertion.component.ts

```diff
@@ -132,6 +132,10 @@ export class PublicBadgeAssertionComponent {
 		return `${assertion.badge.name} - ${assertion.recipient.identity}${fileExtension}`;
 	}
 
+	getFormattedStudyLoad(studyLoad: number): string {
```

Comment:
It would be nice to not have this code repeated so often. Maybe it could be within a helper class?

Reply:
Replaced the function with a pipe

Reply:
Nice!